CODING_CONVENTIONS.md

```diff
@@ -4,6 +4,7 @@
 
 * Code shall be [C11](http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf)
   compliant.
+* Language extensions are permissible if they are implemented on all compilers supported by RIOT.
```

Comment:
We discussed the question of C language extensions in RIOT during the weekly today, and postponed a decision and guidelines to be decided on to the VMA: https://forum.riot-os.org/t/weekly-coordination-call-friday-at-10am-cet/3068/363

I'd therefore propose to defer this PR (or at least the usage of `#include_next` and this change in the coding convention) after a decision has been made.

Reply:
Let's move this to a separate PR then.